### ISSUE TITLE
South Korea - naver.com

### ISSUE THREAD

**Issue by Robinmcgvr** [Robinmcgvr/Kapow-Data#79]
This site is being problematic

**Comment by Robinmcgvr** [comment]
Not sure if this is used for OPR or not

**Comment by Ma-rco** [comment]
yes this is the main site we use for South Korea 

I've noticed some missing days recently - hopefully can get back to normal 🤞🏼

**Comment by Robinmcgvr** [comment]
Those missing days 16-21 March cannot be recovered

**Comment by Ma-rco** [comment]
ok thanks for the update

**Comment by Ma-rco** [comment]
The channels we normally use are Fox, FX and FoxLife. These seemed to have stopped since the beginning of the year - I assume any of this can't be recovered either...?

**Comment by Robinmcgvr** [comment]
Are these channels that changed their names, there has been a lot of this lately?

**Comment by Ma-rco** [comment]
oh I'm not sure if they have or not... I'd have to look into that

**Comment by Ma-rco** [comment]
Yup they all have closed - Fox was replaced by Channel NOW but can't seem to find it on our downloads

**Comment by Robinmcgvr** [comment]
I can't see that channel on the naver site
